libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -476,6 +476,19 @@ type AccountAPI =
                :> ReqBody '[JSON] CompletePasswordReset
                :> MultiVerb 'POST '[JSON] '[RespondEmpty 200 "Password reset successful."] ()
            )
+    :<|> Named
+           "post-password-reset-key-deprecated"
+           ( Summary "Complete a password reset."
+               :> CanThrow 'PasswordResetInProgress
+               :> CanThrow 'InvalidPasswordResetKey
+               :> CanThrow 'InvalidPasswordResetCode
+               :> CanThrow 'ResetPasswordMustDiffer
+               :> Description "DEPRECATED: Use 'POST /password-reset/complete'."
+               :> "password-reset"
+               :> QueryParam' '[Required, Strict, Description "An opaque key for a pending password reset."] "key" PasswordResetKey
```

Comment:
`Caption`, not `QueryParam`.

Now I'm concerned about there not be any tests that cover this change, though.  Maybe better leave this one alone until we remove it?